pkg/api/addons_test.go

```diff
@@ -3749,7 +3768,6 @@ func TestSetAddonsConfig(t *testing.T) {
 				common.IPMASQAgentAddonName,
 				common.AzureCNINetworkMonitorAddonName,
 				common.AzureNetworkPolicyAddonName,
-				common.DNSAutoscalerAddonName,
```

Comment:
Do we need to add `common.CoreDNSAutoscalerName` to this list?

Reply:
Good question. No, that's the name of the _container_, not the _addon_. This change evolves the existing coredns addon. The above UT is looking for addons enabled by default (no change to that set from this PR).